apps/api/src/app/auth/services/passport/github.strategy.ts

```diff
@@ -26,11 +27,12 @@ export class GithubStrategy extends PassportStrategy(githubPassport.Strategy, 'g
 
   async validate(req, accessToken: string, refreshToken: string, profile, done: (err, data) => void) {
     try {
+      const updatedProfile = { ...profile._json, email: profile.emails[0].value };
```

Comment:
Maybe we can change this to `profile`, and change the above profile to `data` or `oauthProfile`

Reply:
Sound like a good upgrade for me.